README.md

```diff
@@ -18,6 +18,8 @@ A DSL is available to configure the task:
 ```groovy
 schemaRegistry {
     url = 'http://localhost:8081/'
+    credentials.username = 'basicauthentication-username' //optional
```

Comment:
Can't we use something like:
```
credentials {
  username = 'something'
  password = 'something else'
}
```
?

Reply:
I would have to create a task and action to support  ```credentials {
  username = 'something'
  password = 'something else'
}``` correct?

Reply:
You should be able to use extensions.
See https://docs.gradle.org/current/userguide/custom_plugins.html#sec:getting_input_from_the_build or https://docs.gradle.org/current/userguide/custom_plugins.html#sec:getting_input_from_the_build

Reply:
added support for 
```
credentials {
  username = 'something'
  password = 'something else'
}
```